invenio_records_resources/resources/record.py

```diff
@@ -35,15 +36,16 @@ class RecordResourceConfig(ResourceConfig):
 class RecordResource(CollectionResource):
     """Record resource."""
 
+    default_config = RecordResourceConfig
+
     def __init__(
         self,
-        config=RecordResourceConfig(),
         service_cls=RecordService,
         *args,
         **kwargs
     ):
         """Constructor."""
-        super(RecordResource, self).__init__(config=config, *args, **kwargs)
```

Comment:
Shouldn't this be something like:

```python
config = config if config else default_config
```

Otherwise where si the logic for "default to `default_config` if no config is passed`. I know it was somewhere, just cannot find it now.

Reply:
Ah you mean this:  https://github.com/inveniosoftware/flask-resources/blob/master/flask_resources/resources.py#L45 right?